Assets/Scripts/RainControl.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections;
+using UnityEngine;
+
+public class RainControl : MonoBehaviour
+{
+    public ParticleSystem rainParticle;
+    public const float RAINCHANCE = 0.3f;
+    private Coroutine stopRoutine;
+
+
+    public void StartRain()
```

Comment:
Let's use this as an easy way to debug the rain :D

Import this from Sirenix.OdinInspector or something

```suggestion
    [Button]
    public void StartRain()
```